tests/course/test_course_pages.py

```diff
@@ -20,33 +23,61 @@ def teacher_course_zero(persist_user):
 
 
 @pytest.fixture
-def authorized_client(client, persist_user):
-    client.force_login(persist_user)
+def persist_profile_teacher_and_student(persist_user):
+    profile = Profile(user=persist_user, account_type='B',
+                      meeting_method='L')
+    profile.save()
+    return profile
+
+
+@pytest.fixture
+def persist_profile_teacher(persist_user):
+    profile = Profile(user=persist_user, account_type='T',
+                      meeting_method='L')
+    profile.save()
+    return profile
+
+
+@pytest.fixture
+def authorized_client_teacher(client, persist_profile_teacher):
+    client.force_login(persist_profile_teacher.user)
+    return client
+
+
+@pytest.fixture
+def authorized_client_teacher_and_student(client, persist_profile_teacher_and_student):
+    client.force_login(persist_profile_teacher_and_student.user)
```

Comment:
Same comment as in this [PR](https://github.com/redhat-beyond/HighTeach/pull/114#discussion_r1216229754)